internal/providers/vault/provider.go

```diff
@@ -54,6 +54,12 @@ func parseVaultID(id string) (string, string) {
 	}
 }
 
+// GetValues takes in variable ids and returns their resolved values. This method is
+// needed to the Provider interface
+func (p *Provider) GetValues(ids ...string) (map[string]plugin_v1.ProviderResponse, error) {
+	return plugin_v1.GetValues(p, ids...)
+}
```

Comment:
Missing docs change in https://github.com/cyberark/secretless-broker/blob/master/internal/providers/vault/README.md

Reply:
This is a new file. I needed to rebase to get this but the changes of this PR have no noticeable impact on the external UX that would warrant a change to this file. Unless if I missed something